.vscode/settings.json

```diff
@@ -3,5 +3,5 @@
   "files.exclude": {
     "**/.git": false
   },
-  "rust-analyzer.check.command": "clippy"
+  "rust-analyzer.check.command": "check"
```

Comment:
why this change?

Reply:
ahh just that my mac was suffering from clippy on save

Reply:
wen new M2 max?